packages/cfd/src/sass/mt5-passwordmodal.scss

```diff
@@ -0,0 +1,226 @@
+.dc-modal__container_cfd-password-modal,
+.dc-mobile-dialog__cfd-password-modal {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    max-width: 424px;
+
+    form {
+        flex-grow: 1;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        justify-content: space-between;
+        padding: 0 2.4rem 1.2rem;
+
+        @include mobile-or-tablet-screen {
+            padding: 0;
+        }
+    }
+    &__description {
+        @include typeface(--small-center-normal-black);
+        color: var(--text-prominent);
```

Comment:
we can pass this color as prop to Text component

Reply:
done 👍